operations/reservations.md

```diff
@@ -1036,6 +1040,8 @@ Adds a new product order of the specified product to the reservation.
 | `ReservationId` | string | required | Unique identifier of the [Reservation](reservations.md#reservation). |
 | `ProductId` | string | required | Unique identifier of the [Product](services.md#product). |
 | `Count` | int | required | The amount of the products to be added. Note that if the product is charged e.g. per night, count 1 means a single product every night. Count 2 means two products every night. |
+| `StartUtc` | string | required | Product start in UTC timezone in ISO 8601 format. |
```

Comment:
Why is it required? That's breaking change, no?

Reply:
Its optional.. my bad fixing it now